statsd/aggregator.go

```diff
@@ -204,13 +214,19 @@ func getContextAndTags(name string, tags []string) (string, string) {
 		sb.WriteString(s)
 	}
 
+	if cardinality.card != "" {
+		sb.WriteString("|card:")
+		sb.WriteString(cardinality.card)
+	}
+
 	s := sb.String()
 
-	return s, s[len(name)+len(nameSeparatorSymbol):]
+	return s, s[len(name)+len(nameSeparatorSymbol) : len(s)-cardStringLen]
```

Comment:
Minute nit, but I think it would be slightly clearer if we had the cardinality field before the tags so the tags are always the last part of the context returned.

```suggestion
	return s, s[len(name)+len(nameSeparatorSymbol) + cardStringLen + len(cardSeparatorSymbol):]
```